src/debug.h

```diff
@@ -78,7 +78,7 @@ void debug_set_exception_handler();
 	These should be put into every thread
 */
 
-#if CATCH_UNHANDLED_EXCEPTIONS == 1
+#if CATCH_UNHANDLED_EXCEPTIONS == 0
```

Comment:
i don't understand why you change this

Reply:
That wasn't supposed to be changed, I only had it like that because I was trying to debug something.

Reply:
Ok it should be fixed now.